ui-tests/test/utils.ts

```diff
@@ -22,6 +22,7 @@ export async function download({
   path: string;
 }): Promise<string> {
   await page.evaluate(async (path: string) => {
+    // TODO Fix this. There is no such thing as passing a `path` here, the filebrowser will download any selected file
```

Comment:
:+1: 

Not sure exactly sure why we had to go this way to download, maybe it was a limitation with Galata.